src/ir_Fujitsu.cpp

```diff
@@ -9,16 +12,6 @@
 #include "IRtext.h"
 #include "IRutils.h"
 
-// Fujitsu A/C support added by Jonny Graham & David Conran
-
-// Equipment it seems compatible with:
-//  * Fujitsu ASYG30LFCA with remote AR-RAH2E
-//  * Fujitsu AST9RSGCW with remote AR-DB1
-//  * Fujitsu ASYG7LMCA with remote AR-REB1E
-//  * Fujitsu AR-RAE1E remote.
-//  * Fujitsu General with remote AR-JW2
```

Comment:
We are actually losing information here if we remove this (as is)
Please add which remote belongs to which A/C in the .h file.
e.g.
```
//  * Fujitsu ASYG30LFCA with remote AR-RAH2E
```
to
```
//   Brand: Fujitsu,  Model: ASYG30LFCA A/C (ARRAH2E)
```

Info from:
https://github.com/crankyoldgit/IRremoteESP8266/blob/af3d2fbefa2261fa3931af680eac0f7884997dd1/src/IRsend.h#L119-L126

This only ever needs to be done if we have "models" supported for the protocol (as in this case) and where we have multiple protocols for a "brand".

It's not a hard & fast requirement (i.e. the formatting) but it's something I've been trying to correct/update etc over time.

Reply:
I might have removed all relevance to each other if the model was already in the section.
In this case the relevant remote is on the line before the A/C.

Adding the Remote in parentheses, with any dash removed is what we want? not better to keep the dash so it can match easier?
https://github.com/crankyoldgit/IRremoteESP8266/blob/master/src/ir_Fujitsu.h#L3-L19

Reply:
Only the first 3 are clear to me which they are
this is the full section, we might have to dig more into commits to see which is what?
```
// Supports:
//   Brand: Fujitsu,  Model: AR-RAH2E remote
//   Brand: Fujitsu,  Model: ASYG30LFCA A/C (ARRAH2E)
//   Brand: Fujitsu,  Model: AR-DB1 remote
//   Brand: Fujitsu,  Model: AST9RSGCW A/C (ARDB1)
//   Brand: Fujitsu,  Model: AR-REB1E remote
//   Brand: Fujitsu,  Model: ASYG7LMCA A/C (ARREB1E)
//   Brand: Fujitsu,  Model: AR-RAE1E remote
//   Brand: Fujitsu,  Model: AGTV14LAC A/C
//   Brand: Fujitsu,  Model: AR-RAC1E remote
//   Brand: Fujitsu,  Model: ASTB09LBC A/C
//   Brand: Fujitsu,  Model: AR-RY4 remote
//   Brand: Fujitsu General,  Model: AR-JW2 remote
//   Brand: Fujitsu,  Model: AR-DL10 remote
//   Brand: Fujitsu,  Model: ASU30C1 A/C
```

Reply:
> I might have removed all relevance to each other if the model was already in the section.
> In this case the relevant remote is on the line before the A/C.
Yeah, but I think the scrape.py sorts them, so any order in the file might be lost.

> Adding the Remote in parentheses, with any dash removed is what we want? not better to keep the dash so it can match easier?
> https://github.com/crankyoldgit/IRremoteESP8266/blob/master/src/ir_Fujitsu.h#L3-L19

I think its better if it matches the enum entries, which they typically should be using.

Reply:
> Only the first 3 are clear to me which they are
> this is the full section, we might have to dig more into commits to see which is what?

No need to dig out more info (just yet). I just didn't want to lose (more) info.

In a later PR, we can update them etc. Keep this PR simple. ;-)